examples/blog-multiple-authors/astro.config.mjs

```diff
@@ -1,13 +1,7 @@
-// Full Astro Configuration API Documentation:
-// https://docs.astro.build/reference/configuration-reference
+import { defineConfig } from 'astro/config';
```

Comment:
It's possible but it would require moving our `bin` script and restructuring the exports a bit. Not sure if I'm comfortable doing all that in a patch.

Reply:
Ah, totally understand. I personally don't mind this approach. Approving 👍

Reply:
See https://github.com/withastro/astro/compare/feat/define-config...feat/define-config-root?expand=1